ctapipe/io/containers.py

```diff
@@ -75,6 +75,10 @@ class RawCameraContainer(Container):
     """
     Storage of raw data from a single telescope
     """
+    event_number = Item(-1, "telescope event number")
+    num_channels = Item(-1, "number of channels per pixel")
```

Comment:
I believe this brings up some good points that need to be addressed. There are some properties that are specific to the GCT camera for which I created a new container which inherits from RawCameraContainer. I then create a new DataContainer class which uses the new containers, and fill this data container in my io file (analogous to hessio.py).

I do this in a separate project to ctapipe for the moment, which is unfortunately inside subversion instead of github so you cannot see what I mean. However I think this is the same idea you have described above.

@kosack is this how we should approach this for different cameras? and should these new containers all be inside containers.py?